readme.md

```diff
@@ -73,6 +73,31 @@ This repository hosts equivalent Go and Python tests written using [snappi](http
     ./do.sh topo rm cpdp
     ```
 
+6. Checking Lint for Python code
+    
+    ```sh
+    # to check whether pyhton code has lint issue
+    ./do.sh pylint
+
+    # we can also specify path to be checked
+    ./do.sh pylint features
+
+    # to format the code according to black 
+    ./do.sh pylint features true
+    ```
+
+7. Checking Lint for Go code
+    
+    ```sh
+    # to check whether go code has lint issue
+    ./do.sh golint
```

Comment:
what if i want to auto format ? we're accepting [true|false] from second argument.

would suggest automatically detecting if we're in actions CI env and raise error (instead of accepting true/false)

Reply:
how do we check the env one thing which is coming to my mind is to be a github action predefined variable but the would restrict it to github actions only 
https://docs.gitlab.com/ee/ci/variables/#:~:text=export%20CI_COMMIT_REF_NAME%3D%22main%22